nameko/standalone/events.py

```diff
@@ -24,50 +21,28 @@ def event_dispatcher(nameko_config, **kwargs):
     """
     amqp_uri = nameko_config[AMQP_URI_CONFIG_KEY]
 
-    kwargs = kwargs.copy()
-    retry = kwargs.pop('retry', True)
-    retry_policy = kwargs.pop('retry_policy', DEFAULT_RETRY_POLICY)
-    use_confirms = kwargs.pop('use_confirms', True)
-    mandatory = kwargs.pop('mandatory', False)
+    serializer = kwargs.pop(
+        'serializer',
+        nameko_config.get(
+            SERIALIZER_CONFIG_KEY, DEFAULT_SERIALIZER
+        )
+    )
+
+    # TODO: standalone event dispatcher should accept context event_data
```

Comment:
Wasn't sure whether to add this as part of this PR. It's much easier to do now, and it would bring it in line with the standalone RPC proxy.